taxi/views.py

```diff
@@ -36,6 +43,19 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
```

Comment:
If you don't use some parameter then better use `_` instead of `*`

```suggestion
    def get_context_data(self, _, object_list=None, **kwargs):
```

Reply:
or if there a lot of parameters than just use `*args` even if you don't use that